templates/repo/home.tmpl

```diff
@@ -30,7 +30,7 @@
 		</div>
 		<div class="gt-df gt-ac gt-fw gt-mt-3" id="repo-topics">
 			{{range .Topics}}<a class="ui repo-topic large label topic" href="{{AppSubUrl}}/explore/repos?q={{.Name}}&topic=1">{{.Name}}</a>{{end}}
-			{{if and .Permission.IsAdmin (not .Repository.IsArchived)}}<button id="manage_topic" class="ui tiny button button-ghost gt-ml-2">{{.locale.Tr "repo.topic.manage_topics"}}</button>{{end}}
+			{{if and .Permission.IsAdmin (not .Repository.IsArchived)}}<button id="manage_topic" class="ui tiny button button-link gt-ml-2">{{.locale.Tr "repo.topic.manage_topics"}}</button>{{end}}
```

Comment:
It can't be link button here.

Other "links" are all labels, but this is only a in-page action, it doesn't navigate, it just shows the form.

Reply:
A ghost button without padding looks weird though. I think we need to find a better place for this button anyways.

Reply:
Think I will just right-align it on the same row as the repo description.

Reply:
But I guess that is for another PR. Can you accept this button here now?

> Other "links" are all labels, but this is only a in-page action, it doesn't navigate, it just shows the form.

I don't understand, this is the first occurence of this new class. What "other" links? I do find this new variane looks better then previous unpadded button-ghost.

Reply:
I prefer to revert  because it is not link.

The problem is that at the moment Gitea's UI framework is still far from complete.

Other framework's  "ghost button" by default has some margin/padding styles, and Fomantic UI's tertiary also inherits some styles from "ui button".

But you want to use some "plain buttons" without style, without a careful and clear design, eventually there are conflicts and abuses.

Reply:
ps: in most cases, I think it needs a "plain" button , but not a "ghost" button.

A plain button should have no hover background/margin/padding, then every problem is resolved, and no need to do any style overriding or `gt-` patching.

I think most "button-ghost" usages in code applies to such style (no hover background), that's why I suggested (preferred) the "button-plain" name.

Reply:
I reverted these `button-link` changes for now. I see the button system itself needs to be more fleshed out later.